aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -1346,6 +1358,7 @@ TORCH_LIBRARY_IMPL(_quantized, QuantizedCPU, m) {
 TORCH_LIBRARY_IMPL(quantized, CPU, m) {
   m.impl(TORCH_SELECTIVE_NAME("quantized::linear_with_input_q_dq_qweight_dq_output_fp32"), TORCH_FN(QLinearInt8FusedQDQ<false>::run));
   m.impl(TORCH_SELECTIVE_NAME("quantized::linear_with_input_q_dq_qweight_dq_relu_output_fp32"), TORCH_FN(QLinearInt8FusedQDQ<true>::run));
+  m.impl(TORCH_SELECTIVE_NAME("quantized::int4mm_packed_weight_cpu"), TORCH_FN(at::native::_weight_int4pack_mm_cpu_tensor));
```

Comment:
cc @drisspg this `_cpu` is not recommended right? I remember you mentioned this in some post

Reply:
Thanks for the comment. Do you mean to remove `_cpu` or to change it to `_for_cpu`?

Reply:
no, I was thinking that tensor.device should be used to distinguish between cuda and cpu, but somehow sdpa is hacking this and used `_for_cpu` in the op, which is copied by other ops like this